### PR TITLE
QtTestBrowser SSL error confirmation

### PR DESCRIPTION
**Pull request by ObserverHerb**
We use this internally for testing/configuration and I'd like to contribute it back.

### CODE REVIEW

**Comment by annulen** [comment]
Could you retarget it to qtwebkit-stable please?

**Comment by herbmillerjr** [comment]
Done.

**Comment by annulen** [comment]
Move slot to LauncherWindow

File: Tools/QtTestBrowser/mainwindow.cpp [hunk 1]
```diff
@@ -249,3 +252,19 @@ void MainWindow::onTitleChanged(const QString& title)
...
+        errorStrings += "<li>" + error.errorString() + "</li>";
+    errorStrings += "</ul>";
+
+    QMessageBox sslWarningBox;
+    sslWarningBox.setText("SSL handshake problem");
+    sslWarningBox.setInformativeText(errorStrings);
+    sslWarningBox.setStandardButtons(QMessageBox::Abort | QMessageBox::Ignore);
+    sslWarningBox.setDefaultButton(QMessageBox::Abort);
+    sslWarningBox.setIcon(QMessageBox::Warning);
+    if (sslWarningBox.exec() == QMessageBox::Ignore) reply->ignoreSslErrors();
```
Comment (annulen) [inline Tools/QtTestBrowser/mainwindow.cpp]: Line break is needed after if()

**Comment by annulen** [comment]
Changes pushed to qtwebkit-stable, thanks for your contribution!

**Pull request closed without merging**